src/files/coverageparser.ts

```diff
@@ -54,11 +54,12 @@ export class CoverageParser {
     }
 
     private async convertSectionsToMap(
+        coverageFilename: string,
         data: Section[],
     ): Promise<Map<string, Section>> {
         const sections = new Map<string, Section>();
         const addToSectionsMap = async (section: Section) => {
-            sections.set(section.title + "::" + section.file, section);
+            sections.set([coverageFilename, section.title, section.file].join("::"), section);
```

Comment:
I am curious how this will decide between two conflicting coverage files / lines? Do we just need to assume the "best case" scenario and pick the coverage line that is more positive?